src/test/java/org/jabref/model/entry/EntryLinkListTest.java

```diff
@@ -0,0 +1,45 @@
+package org.jabref.model.entry;
+
+import org.jabref.model.database.BibDatabase;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Optional;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertEquals;
+
+public class EntryLinkListTest {
+
+    private String key = "test";
+    private BibDatabase database = new BibDatabase();
+    private List<ParsedEntryLink> links = EntryLinkList.parse(key, database);
+    private ParsedEntryLink link = links.get(0);
+
+    @Test
+    public void givenFieldValueAndDatabaseWhenParsingThenExpectKey() {
+        assertEquals(key, link.getKey());
+    }
+
+    @Test
+    public void givenFieldValueAndDatabaseWhenParsingThenExpectDataBase() {
+        assertEquals(database, link.getDataBase());
+    }
+
+    @Test
+    public void givenFieldValueAndDatabaseWhenParsingThenExpectEmptyLinkedEntry() {
```

Comment:
Is it possible to add a "real" test linking a "real" other entry?

Reply:
I do not understand. Is a "real" other entry just a `new BibEntry()` for you?